components/swatch/stories/template.js

```diff
@@ -31,11 +35,19 @@ export const Template = ({
 		case "none":
 			borderStyle = "noBorder";
 			break;
+		case "border":
+			borderStyle = "border";
+			break;
 		case "light":
 			borderStyle = "lightBorder";
 			break;
 	}
```

Comment:
Note to self, this is a great place to align more closely with the web components API in future